pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -173,6 +173,12 @@
   SSL_FACTORY_ARG("sslfactoryarg", null,
       "Argument forwarded to constructor of SSLSocketFactory class."),
 
+   /**
+    * The String argument to give to the constructor of the SSL Context
+    */
+   SSL_FACTORY_PROTO("sslContextProtocol", "TLS",
+          "Argument forwarded to constructor of SSLContext class and is used to specify the SSL Context algo used."),
```

Comment:
I suggest renaming SSL_FACTORY_PROTO to SSL_CONTEXT_PROTOCOL to match up with the text of the property.

Also, the description should be updated to reference "protocol" as it still references algo\[rithm\]. Maybe something like: "Explicit SSL protocol to use when creating an SSLContext."